scandium-core/src/main/java/org/eclipse/californium/scandium/dtls/Handshaker.java

```diff
@@ -1239,6 +1262,50 @@ public final void handshakeFailed(Throwable cause) {
 		SecretUtil.destroy(this);
 	}
 
+	/**
+	 * Abort handshake.
+	 * 
+	 * Notifies all registered session listeners about a handshake failure.
+	 * Listeners are intended to keep the connection.
+	 * 
+	 * If {@link #setFailureCause(Throwable)} was called before, only calls with
+	 * the same cause will notify the listeners. If
+	 * {@link #setFailureCause(Throwable)} wasn't called before, sets the
+	 * <em>cause</em> property to the given cause.
+	 * 
+	 * @param cause The reason for the abort.
+	 * @see #handshakeFailed(Throwable)
+	 * @see #isRemovingConnection()
+	 */
+	public final void handshakeAborted(Throwable cause) {
```

Comment:
I do not understand the benefits of this `final` ?

Reply:
The api-changes.json contains:

"justification": "keep API consistent with other similar functions" 

FMPOV, similar functions are:
[handshakeCompleted](https://github.com/eclipse/californium/blob/master/scandium-core/src/main/java/org/eclipse/californium/scandium/dtls/Handshaker.java#L1205) or [handshakeFailed(Throwable cause)](https://github.com/eclipse/californium/blob/master/scandium-core/src/main/java/org/eclipse/californium/scandium/dtls/Handshaker.java#L1224), both are already final.

Reply:
Without API break in mind : 
I don't understand the benefits of all those `final` keyword. Most of the time I really don't like the use of `final` for methods or classes.

With API break in mind :
Even if I agree that `finalMethodAddedToNonFinalClass` is a questionable break, I would not break just for this, mainly because I find `final` keyword for method a bad idea most of the time.